### PR TITLE
homepage, blogs and organisation frontend

### PR DESCRIPTION
**Pull request by ferhatu**

### CODE REVIEW

File: client/src/components/Organisations.js [hunk 1]
```diff
@@ -1,130 +1,44 @@
...
-  const filteredNames = (value) => {
-    setOrgs(
-      data.filter((org) =>
-        (org.name.toLowerCase() || org.category.toLowerCase()).includes(value)
-      )
-    );
+  const [org, setOrg] = useState([]);
+  const [searchOrg, setSearchOrg] = useState([]);
+  useEffect(() => {
+    fetch("http://localhost:5000/api/organisations/org")
```
Comment (ahmad-ali14) [inline client/src/components/Organisations.js]: always fetch domain, this will not work on heroku

File: client/src/components/Blogs.js [hunk 2]
```diff
@@ -0,0 +1,261 @@
+import React from "react";
+
+const Blogs = () => {
```
Comment (ahmad-ali14) [inline client/src/components/Blogs.js:3]: u mean resources?

**Review by minwwgo — APPROVED** [review]

**Review by 137rx — APPROVED** [review]

File: client/src/components/Blogs.js (hunk 2 above)
Comment (ferhatu) [reply client/src/components/Blogs.js:3]: it is blogs page which shows popular websites.

File: client/src/components/Organisations.js (hunk 1 above)
Comment (ferhatu) [reply client/src/components/Organisations.js]: I changed this part

**Pull request merged**